### PR TITLE
Nit fixes to releasing doc

### PR DESCRIPTION
**Pull request by cijothomas**
Added example for tagging
Fixed some markdown violations suggestion by VSCode plugins.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -24,24 +24,33 @@ A draft PR can be created, but before releasing consider the following:
...
-
 ## Tagging Convention
 
 For each crate: it should be `<crate-name>-<version>` `<version>` being the simple `X.Y.Z`.
+For example:
+
+```sh
+git tag -a opentelemetry-http-0.11.1 -m "opentelemetry-http 0.11.1 release"
+git push origin opentelemetry-http-0.11.1
+```
```
Comment (TommyCpp) [inline RELEASING.md:53]: Or using Github UI to tag

**Review by TommyCpp — APPROVED** [review]

File: RELEASING.md (hunk 1 above)
Comment (cijothomas) [reply RELEASING.md:53]: I used command line, as I am sure this is something we can automate, when the time comes.

**Comment by hdost** [comment]
> VSCode plugins

What plugins ?

**Comment by cijothomas** [comment]
> > VSCode plugins
> 
> What plugins ?

[DavidAnson.vscode-markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)

**Pull request merged**